client/src/components/Main/Main.jsx

```diff
@@ -11,20 +9,24 @@ import SurveyDetail from '../../pages/SurveyDetail';
 import TakeSurvey from '../../pages/TakeSurvey';
 
 const Main = () => {
-  // const { auth } = useSelector((state) => state.adminLoginReducer);
-  // const snackbar = useSelector((state) => state.snackbarReducer);
+  const { data } = useSelector((state) => state.adminLoginReducer);
+  const { auth } = data;
```

Comment:
this can be simplified to `const {data: {auth}} = useSelector(...)`, though this is arguably less readable.

Reply:
Could be done this way but I agree with your comment of less readable!